pages/docs/balance/business-verified-customer/handle-verification-statuses.mdx

```diff
@@ -780,21 +790,21 @@ If the document was found to be fraudulent or doesn’t match the identity of th
+| `BusinessTypeMismatch`    | Business type chosen does not match document                           | Based on the documentation provided, the entity type for this business should be created as a/an (LLC, Corporation, Sole Prop). The account has been moved to [retry](#handling-status-retry) so that the correct business type (LLC, Corporation, Sole Prop etc.) can be submitted.                                 |
+| `ScanDobMismatch`         | Scan DOB does not match DOB on account                                 | The DOB listed on the ID uploaded does not match the DOB on the user's account. The account has been placed in [retry](#handling-status-retry) so that the user can adjust the DOB listed on the account to match the ID which was provided.                                                                         |
+| `ScanFailedOther`         | ID may be fraudulent or a generic example ID image                     | The ID uploaded may be fraudulent or a generic example of an ID. The user needs to upload an valid ID to proceed with account verification.                                                                                                                                                                          |
+| `ScanIdExpired`           | ID is expired or missing expiration date                               | The ID uploaded by the user is expired. The user will need to upload a non-expired ID.                                                                                                                                                                                                                               |
+| `ScanIdTypeNotSupported`  | ID may be a military ID, firearm license, or other unsupported ID type | The uploaded ID is not an acceptable form of ID. [Here](#acceptable-documents) is a list of ID types that Dwolla accepts for account verification.                                                                                                                                                                   |
+| `ScanIdUnrecognized`      | ID is not recognized                                                   | The ID which has been uploaded is unreadable. The user will need to upload a new image of their ID to proceed with verification.                                                                                                                                                                                     |
+| `ScanNameMismatch`        | Scan name does not match name on account                               | The name listed on the ID which has been uploaded by the user does not match the name which is listed at the account level. The account has been placed in [retry](#handling-status-retry) so that the user can adjust the name on the account to match the ID which was provided.                                   |
```

Comment:
```suggestion
| `ScanNameMismatch`        | Scan name does not match name on account                               | The name listed on the ID which has been uploaded by the user does not match the name which is listed at the account level. The account has been placed in [retry](#handling-retry-status) so that the user can adjust the name on the account to match the ID which was provided.                                   |
```